cursor.go

```diff
@@ -72,7 +74,6 @@ type Cursor struct {
 }
 
 func (c *Cursor) Each(f func(*RowBatch) error) error {
-	c.lastSuccessfulPrimaryKey = 0
```

Comment:
what was this here for to begin with?

Reply:
It was to ensure that subsequent calls to `.Each` would start from scratch. It turns out we don't actually use this method in this way.